2023-08-11 16:50:40,338 INFO    MainThread:68855 [internal.py:wandb_internal():90] W&B internal server running at pid: 68855, started at: 2023-08-11 16:50:40.337835
2023-08-11 16:50:40,339 DEBUG   HandlerThread:68855 [handler.py:handle_request():141] handle_request: run_start
2023-08-11 16:50:40,339 INFO    WriterThread:68855 [datastore.py:open_for_write():75] open: /home/nicole/marl-emecom/src/experiments_anastassacos/wandb/offline-run-20230811_165039-10xebl0b/run-10xebl0b.wandb
2023-08-11 16:50:40,341 DEBUG   HandlerThread:68855 [meta.py:__init__():35] meta init
2023-08-11 16:50:40,341 DEBUG   HandlerThread:68855 [meta.py:__init__():49] meta init done
2023-08-11 16:50:40,341 DEBUG   HandlerThread:68855 [meta.py:probe():209] probe
2023-08-11 16:50:40,342 DEBUG   HandlerThread:68855 [meta.py:_setup_git():199] setup git
2023-08-11 16:50:40,350 DEBUG   HandlerThread:68855 [meta.py:_setup_git():206] setup git done
2023-08-11 16:50:40,351 DEBUG   HandlerThread:68855 [meta.py:_save_pip():53] save pip
2023-08-11 16:50:40,351 DEBUG   HandlerThread:68855 [meta.py:_save_pip():67] save pip done
2023-08-11 16:50:40,351 DEBUG   HandlerThread:68855 [meta.py:probe():247] probe done
2023-08-11 16:50:48,379 WARNING MainThread:68855 [internal.py:wandb_internal():153] Internal process interrupt: 1
2023-08-11 16:50:48,573 WARNING MainThread:68855 [internal.py:wandb_internal():153] Internal process interrupt: 2
2023-08-11 16:50:48,573 ERROR   MainThread:68855 [internal.py:wandb_internal():156] Internal process interrupted.
2023-08-11 16:50:49,344 INFO    SenderThread:68855 [sender.py:finish():1111] shutting down sender
2023-08-11 16:50:49,389 INFO    WriterThread:68855 [datastore.py:close():279] close: /home/nicole/marl-emecom/src/experiments_anastassacos/wandb/offline-run-20230811_165039-10xebl0b/run-10xebl0b.wandb
2023-08-11 16:50:49,390 INFO    HandlerThread:68855 [handler.py:finish():806] shutting down handler
2023-08-11 16:50:49,391 INFO    MainThread:68855 [internal.py:handle_exit():80] Internal process exited
